static/app/utils/profiling/profile/sampledProfile.tsx

```diff
@@ -42,6 +43,7 @@ export class SampledProfile extends Profile {
       });
 
       if (
+        options.type === 'flamechart' &&
```

Comment:
This feels awfully error prone if we have to check if we're in flamechart/flamegraph mode in all the formats

Reply:
Yeah, agree. The other option is to create a different primitive to import these, but that's more work than I'd like to do right now